angular-rails-app/app/controllers/dob_permits_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DobPermitsController < ApplicationController
+
+  def index 
+    if params[:property_id]
+      @dob_permits = DobPermit.where(
```

Comment:
Will definitely keep this in mind!  I think I'm going to leave this particular one as is for the moment, as I anticipate these action blocks will actually be different.  I would like to add some `group by` and `order by` to the queries, which will be different for each Model (or may not get added at all for some Models). However, since many of the tables are missing data, I waited on that part.